README.md

```diff
@@ -33,19 +33,15 @@ This will create a local copy of the [Apostrophe 3 boilerplate](https://github.c
 
 Use the `--a2` flag when creating an Apostrophe 2 project to use [the boilerplate](http://github.com/apostrophecms/apostrophe-boilerplate) for that version.
 
-#### `--boilerplate`
+#### `--repo`
```

Comment:
This could be misunderstood as the name of their own repo for their project, which will be a thing. I don't see what's wrong with boilerplate.

Reply:
This was included in the #2 proposal. We could use `starter` instead. `boilerplate` is used more for a single piece of code in most places I see. "Starter" seems to be the common term. Gatsby has a "Starter library," the classic Underscore WP theme is called a starter, etc.